packages/server/lib/modes/interactive.ts

```diff
@@ -170,6 +173,36 @@ export = {
       makeGraphQLServer(),
     ])
 
+    const clearCtxAndQuit = async () => {
```

Comment:
This file contains the bulk of the changes. Most of the other changes in this diff are related to `await`ing function calls that are now async